code/index.html

```diff
@@ -3,28 +3,131 @@
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
     <title>Project 3</title>
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
+    
     <!-- video or image as a header is cool :) -->
+    <header class="hero-image">
+      
+      <div class="inner-hero-header"> 
+        <nav class="navbar">
+          
+          <a href="#" class="logo">Frusto</a>
+          <ul class="nav-menu">
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">About us</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">News</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Contact</a>
+            </li>
+          
+          </ul>
+          </nav>
+      </div>
+
+      
+      <div class="logo-place">
+        <a href="#" class="logo-mobil">Frusto</a>
+      </div>
+      
+    </header> 
 
+  <main class="main-container">
+    
     <!-- Signup form -->
+    <section class="form-wrapper">
+      <h2>Sign up and start unloading your feelings</h2> 
+      <form id="form" class="form-inputs" action="https://httpbin.org/anything" method="post"> 
+      
+        <div class="textarea">
+          <label for="fname">First name</label>
+          <input id="fname" type="text" name="fname"/>
+        
+          <label for="lname">Last name</label>
+          <input id="lname" type="text" name="lname" />
+
+          <label for="email">email</label>
+          <input id="email" type="email" name="email"/>
+        
+          <label for="password">Password</label>
+          <input id="password" type="password" name="password" required/>
+        </div>
+      
+   
+        <div class="radiobutton">
+          <div>
+            <input id="prof" type="radio" name="occupation" value="prof"/>
+            <label for="prof">young and restless</label>
+          </div>
+        </div>
+        
+        <div>
+          <input id="student" type="radio" name="occupation" value="student"/>
+          <label for="student">Tired and frustrated</label>
+        </div>
+      
+        <div class="checkbox">
+          <input id="agreement" type="checkbox" name="agreement"/>
+          <label for="agreement">Agree to the terms and conditions.</label>
+          <br>
+          <input id="newsletter" type="checkbox" name="news"/>
+          <label for="newsletter">Subscribe to the newsletter.</label>
+        </div>
+      
+        <button id="submit" type="submit" name="submit">SIGNUP NOW</button>
+  
+      </form>
+    </section> 
+
 
-    <section class="accordion">
-      <!-- Accordion -->
+    <!-- Accordion -->
+    <section class="accordion-container">
+    
```

Comment:
I was missing a bit of a header here for the introduction to your business - just a personal choice and not a critique of your code in any way.